src/libsystemd/sd-event/sd-event.c

```diff
@@ -1868,9 +1868,7 @@ _public_ int sd_event_trim_memory(void) {
 
         log_debug("Memory pressure event, trimming malloc() memory.");
 
-#if HAVE_GENERIC_MALLINFO
-        generic_mallinfo before_mallinfo = generic_mallinfo_get();
-#endif
+        struct mallinfo2 before_mallinfo = mallinfo2();
```

Comment:
this is not guaranteed to exist, so the generic infrastructure needs to remain, for older glibc

Reply:
ok, missed that